docs/application/native/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,258 @@
+
+- Managing component context
+
+  You can retrieve the running component context and operate on it. The `component_context_h` handle is related to the running components and can be used to [manage the component context](#manage_context).
+
+  For more information on the functions that use the `component_context_h` handle, see the Component Context API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) applications).
+
+- Getting component information
+
+  The `component_info_h` handle is related to the available components that are installed but not necessarily running.
+
+  For more information on the functions that use the `component_info_h` handle, see the Component Information API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__INFO__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__INFO__MODULE.html) applications).
+
+Iterator functions are used to travel through a list of components. `component_manager_foreach_component_context()` is used in running components and `component_manager_foreach_component_info()` is used in available components. Each function calls a callback function (`component_manager_component_context_cb()` or `component_manager_component_info_cb()`), passing the handle for each component.
+
+## Prerequisites
+
+1. To use the functions and data types of the Component Manager API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+      <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+      <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+    </privileges>
+    ```
+
+2. Include the `<component_manager.h>` header file in your application:
+
+    ```
+    #include <component_manager.h>
+    ```
+<a name="manage_context"></a>
+## Managing Component Context
+
+To get the running component context and its details, and to operate on the context:
+
+1. Get the context of the currently running component with `component_manager_get_component_context()`. Its parameters are the ID of the component from which the context is being obtained, and the handle (`component_context_h`) to the component context which is filled with the received context.
+
+   To get a component's context, the component must be running.
+   ```
+   component_context_h component_context = NULL;
+   int ret = component_manager_get_component_context(`Your Component ID`, &component_context);
+   ```
+   If the function returns `COMPONENT_MANAGER_ERROR_NONE`, it has executed correctly and the `component_context` variable now contains the handle to the defined component context.
```

Comment:
Suggestion : 
If it executed correctly, the function returns `COMPONENT_MANAGER_ERROR_NONE` and `component_context` variable contains the handle to the defined component context.

Reply:
Done